pom.xml

```diff
@@ -107,8 +107,23 @@
         <contributor>
             <name>Niall Pemberton</name>
         </contributor>
+        <contributor>
+            <name>Arturo Bernal</name>
+        </contributor>
     </contributors>
 
+    <dependencyManagement>
```

Comment:
@arturobernalg 
Why? This looks like a duplication of the parent POM. What am I missing?